cms/templates/js/content-group-details.underscore

```diff
@@ -1,6 +1,6 @@
-<div class="wrapper-group-configuration">
-    <header class="group-configuration-header">
-        <h3 class="group-configuration-title">
+<div class="collection-details wrapper-group-configuration">
+    <header class="collection-header group-configuration-header">
```

Comment:
Can remove wrapper-group-configuration, group-configuration-header, and group-configuration-title.